scss/techs.scss

```diff
@@ -0,0 +1,46 @@
+@import 'colors';
+
+.main-tech-container {
+  position: relative;
+}
+
+.background-tech-container {
+    border: 1px solid;
+    border-color: $green;
+    border-radius: 5px;
+    height: 300px;
```

Comment:
As both card have the same width and height I would extract both of them into variables, and then reference them inside the class block
```
$card-height: 300px;
$card-width: 200px;

.background-tech-container {
    height: $card-height;
    width: $card-width;
}
```

The same for `.data-tech-container`

Reply:
Done